CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+## Types of Contributions
+
+### Report Bugs
+
+Report issues and bugs at https://git.equinor.com/fmu-utilities/fmu-config/issues.
+
```

Comment:
This is incorrect. Shall be `github.com/equinor/fmu-config`

Reply:
Thanks, missed this one